skaffold.yaml

```diff
@@ -84,5 +84,12 @@ deploy:
         wait: true
         chartPath: charts/admission
         setValueTemplates:
-          runtime.image.repository: '{{.IMAGE_REPO_europe_docker_pkg_dev_gardener_project_public_gardener_extensions_registry_cache_admission}}'
-          runtime.image.tag: '{{.IMAGE_TAG_europe_docker_pkg_dev_gardener_project_public_gardener_extensions_registry_cache_admission}}@{{.IMAGE_DIGEST_europe_docker_pkg_dev_gardener_project_public_gardener_extensions_registry_cache_admission}}'
+          global.image.repository: '{{.IMAGE_REPO_europe_docker_pkg_dev_gardener_project_public_gardener_extensions_registry_cache_admission}}'
+          global.image.tag: '{{.IMAGE_TAG_europe_docker_pkg_dev_gardener_project_public_gardener_extensions_registry_cache_admission}}@{{.IMAGE_DIGEST_europe_docker_pkg_dev_gardener_project_public_gardener_extensions_registry_cache_admission}}'
+profiles:
+- name: remote-extensions
+  patches:
+  - op: add
+    path: /deploy/helm/releases/0/setValues
+    value:
+      global.vpa.enabled: false
```

Comment:
Why we need this?

Gardener does not support Seeds without VPA: https://github.com/gardener/gardener/blob/901e117ffa2ea5c9c08f764fb3d37aa239a0db5f/docs/operations/seed_settings.md#L111
Hence, I don't see why we need to disable VPA for the extension.

```suggestion
```

Reply:
This is for the admission controller which is deployed in the garden cluster where we do not have VPA and hence VPA crds.

Reply:
I see. In the existing local setup this is not an issue because the garden cluster (kind) is registered also as Seed and VPA is installed in this way. But we don't have this in the provider-extensions setup. Thanks for clarifying!